### ISSUE TITLE
Titles get capped on import via WoS

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backend#685]
# Bug

Titles get capped on import

## Context

Import with WoS file:
https://www.webofscience.com/wos/woscc/full-record/WOS:000332386100024

Title in WoS txt file:
<img width="502" alt="Schermafbeelding 2022-10-11 om 17 00 43" src="https://user-images.githubusercontent.com/1572885/195127325-a54dd284-6ab2-4aad-b322-b44dc3978ce6.png">

See import:
![Image](https://user-images.githubusercontent.com/1572885/195021771-563b88a7-5df8-455a-b5ae-082d4b341a8f.png)

See full title on copernicus:
https://acp.copernicus.org/articles/14/2155/2014/

**Comment by mietcls** [comment]
@netsensei Ha, I found out why – it is not importing it via DOI, but importing via WoS.

When I download the record via here https://www.webofscience.com/wos/woscc/full-record/WOS:000332386100024

I can see that in the txt file the title is on multiple lines. This causes the field to break I think.
<img width="522" alt="Schermafbeelding 2022-10-11 om 16 58 24" src="https://user-images.githubusercontent.com/1572885/195126699-5ca93f51-b88f-43c5-9efc-5c70281ff896.png">

Tested with multiple types of downloads, it all behaves the same.
